ansible/extra_vars.yml

```diff
@@ -18,37 +18,4 @@ target_platform: local
 experimental_features: disable
 
 # Variable used to tell to the playbook if we will install or not istio
-service_mesh: true
-
-# Variables used to override or configure the Istio playbook
-
-# Istio github branch to be used to install the istio playbook
-istio_git_branch: master
-
-# Folder where the project will be cloned on your machine
-istio_repo_dest: ~/.istio/playbooks
-
-# Istio playbooks variables - see Istio Ansible README file for more info
-cluster_flavour: ocp
-
-# The GitHub token to use when calling GitHub's API
-# The token is needed in order to avoid getting Rate Limited by GitHub's API
-# Rate Limiting can easily occur in large corporate environments
-# that make outgoing calls through a small range of IPs
-# If the token is not set, then no API Token will be used
-github_api_token: ""
-
-istio:
-  release_tag_name: "0.6.0"
```

Comment:
This var should be then added to the default value of the role ?

Reply:
The `github_api_token` is not needed at all. The `release_tag_name` was added in `ansible/playbook/roles/install_istio/vars/main.yml`

Reply:
But I override the value within the `extra_vars.yml` file. So keep it please like the dest folder which is different.

Reply:
I have no problem changing things back to the way they were, but I'm trying to understand why we would need to.
I might be missing something, but why would we need it in `extra_vars` when it's already been defined in `vars/main.yml` that I have added? The benefit of having it in `vars/main.yml` is that the role works even without having to specify the `extra_vars` file.
Furthermore, why does the `dest` folder need to be set specifically to `/home/istio`?

Reply:
> dest folder

it is used to install `istio` distro containing `istioctl` client, examples, ...

Reply:
Yes I know, my question was basically whether it matters were it's installed?

I'll go ahead and change it back :)

Reply:
Changed in `vars/main.yml`